### PR TITLE
Added checkers for integers, floats and decimal

### PR DESCRIPTION
**Pull request by j0nimost**
Added new static methods for the `Check` class;

- Positive
- Range

### CODE REVIEW

File: framework/src/Volo.Abp.Core/Volo/Abp/Check.cs [hunk 1]
```diff
@@ -162,4 +162,182 @@ public static ICollection<T> NotNullOrEmpty<T>(ICollection<T> value, [InvokerPar
 
         return value;
     }
+
+    public static Int16 NotNegativeOrZero(
```
Comment (hikalkan) [inline framework/src/Volo.Abp.Core/Volo/Abp/Check.cs]: Please rename all to `Positive`

File: framework/src/Volo.Abp.Core/Volo/Abp/Check.cs [hunk 2]
```diff
@@ -162,4 +162,182 @@ public static ICollection<T> NotNullOrEmpty<T>(ICollection<T> value, [InvokerPar
...
+        else if (type < 0)
+        {
+            throw new ArgumentException($"{parameterName} is less than zero");
+        }
+        return type;
+    }
+
+
+
+    public static Int16 WithinRange(
```
Comment (hikalkan) [inline framework/src/Volo.Abp.Core/Volo/Abp/Check.cs]: Please rename all to `Range`

**Review by hikalkan — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please rename the methods as I suggested. Also, add unit tests for these methods.

**Comment by rqx110** [comment]
rename param name `type` to `value`?
e.g. `Int16 type`  --> `Int16 value`

File: framework/src/Volo.Abp.Core/Volo/Abp/Check.cs (hunk 2 above)
Comment (j0nimost) [reply framework/src/Volo.Abp.Core/Volo/Abp/Check.cs]: I have implemented the change  in this commit: [aa08d0d](https://github.com/abpframework/abp/pull/12576/commits/aa08d0d5ad86e298e433c15c26a82a36f8a173b9)

File: framework/src/Volo.Abp.Core/Volo/Abp/Check.cs (hunk 1 above)
Comment (j0nimost) [reply framework/src/Volo.Abp.Core/Volo/Abp/Check.cs]: I have implemented the change  in this commit: [aa08d0d](https://github.com/abpframework/abp/pull/12576/commits/aa08d0d5ad86e298e433c15c26a82a36f8a173b9)

**Comment by j0nimost** [comment]
> rename param name `type` to `value`? e.g. `Int16 type` --> `Int16 value`

I have renamed the property names for all in this commit:  [daab524](https://github.com/abpframework/abp/pull/12576/commits/daab524943f7e6bad8259fa6a90ee833495236f1)

**Review by hikalkan — APPROVED** [review]

**Pull request merged**